src/main/kotlin/app/revanced/patches/youtube/layout/forceolduilayout/bytecode/fingerprints/ForceOldUILayoutFingerprint.kt

```diff
@@ -0,0 +1,19 @@
+package app.revanced.patches.youtube.layout.forceolduilayout.bytecode.fingerprints
+
+import app.revanced.patcher.extensions.or
+import app.revanced.patcher.fingerprint.method.impl.MethodFingerprint
+import org.jf.dexlib2.AccessFlags
+import org.jf.dexlib2.Opcode
+
+object ForceOldUILayoutFingerprint : MethodFingerprint(
+    "L", AccessFlags.PUBLIC or AccessFlags.STATIC, listOf("L"), listOf(
+        Opcode.IGET_OBJECT,
+        Opcode.GOTO,
+        Opcode.CONST_STRING,
+    ),
+
+    // Instead of applying a bytecode patch, it might be possible to use only integration java code and
+    // manually set the desired version string as this keyed value in the SharedPreferences.
+    // But, this bytecode patch is simple and it works.
+    strings = listOf("pref_override_build_version_name")
```

Comment:
This might cause unintended behaviour. Technically this patch only allows to force the old UI over abusing this override, but does not convey that it might change other things. Should this patch not be called what it does? While it forces the old UI as a side effect, its doing something else which is `spoof-build-version-name`. The patch description can then explain what side effects it causes such as bring back the old layout.

Reply:
Yeah I think that's better.  Although I think 'spoof' or some other wording is better than 'override', because it may confuse the user to think the patch uses an older version of YouTube (when it's really just spoofing what YouTube thinks what version it is)

How should the settings present the disclaimer?  It's a lot to stick something like this into a summary title:
"When switching off, the old layout may persist until user is logged out or app data is cleared"

Reply:
Maybe call the patch "SpoofOldAppVersionPatch"?

Reply:
> Maybe call the patch "SpoofOldAppVersionPatch"?

In that case, it should be called `spoof-app-version`. The patch is spoofing the _app version_ **to** the old version and does not spoof the _old_ app version.

Reply:
> How should the settings present the disclaimer?

The description can be wordy, though it would break our current efforts of them being short. I don't have another solution yet, for that reason using a wordy description is fine. If possible, also open an issue for this for further reference after merge.